addons/point_of_sale/static/src/app/screens/receipt_screen/receipt/receipt.js

```diff
@@ -42,4 +42,22 @@ export class OrderReceipt extends Component {
             !(line.display_discount_policy == "without_discount" && line.price < line.price_lst)
         );
     }
+    /**
+     * Override this method to customize the tax letter mapping.
+     * @returns {Object} mapping of tax amount to tax letter
+     * @example
+     * return {
+     *    0: "A",
+     *    5: "B",
+     *    12: "C",
+     *    18: "D",
+     * };
+     */
+    getTaxLetterMapping() {
+        return {};
+    }
+
+    getTaxLetter(taxAmount) {
```

Comment:
I think it's safer to assume that tax-letter mapping will be dependent on the whole tax object, not only on its tax amount. Redo this such that `getTaxLetter()` takes the tax object. Be default, it will just return an empty string. And other addons will override `getTaxLetter` to return the correct letter based on the tax record (object).

With this, we can get rid of the `getTaxLetterMapping` function.

(Am I inconsistent? Apology in advance.)